source/index.txt

```diff
@@ -73,104 +106,96 @@ suit your needs.
 
    * -
      - Compass
-     - Compass Community
-     - Compass Readonly
      - Compass Isolated
+     - Compass Readonly
+     - Compass Community
 
-   * - View :doc:`documents </documents>`,
-       :doc:`collections </collections>`,
-       :doc:`databases </databases>`, and :ref:`indexes <indexes-tab>`
+   * - Read data from your MongoDB database, including visual query
+       and aggregation builders.
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
-   * - Interact with documents, collections, and databases with full
-       CRUD functionality
+   * - Create, update, or delete data from your MongoDB database
      - |checkmark|
      - |checkmark|
-     -
+     - 
      - |checkmark|
 
-   * - Build and run ad hoc queries
+   * - View indexes and related statistics
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
 
-   * - Manage indexes: view stats, create, and delete
+   * - Create or Delete Indexes
      - |checkmark|
      - |checkmark|
-     -
+     - 
      - |checkmark|
 
    * - View existing :ref:`document validation rules <validation>`
      - |checkmark|
-     - 
      - |checkmark|
      - |checkmark|
+     - 
 
-   * - Create, delete and edit document validation rules
+   * - Modify or remove document validation rules
      - |checkmark|
-     - 
-     -
      - |checkmark|
+     - 
+     - 
 
    * - View and optimize query performance with visual explain plans
      - |checkmark|
      - |checkmark|
-     -
+     - 
      - |checkmark|
 
-   * - Create and execute aggregation pipelines
+   * - Support for Clusters enforcing Kerberos, LDAP, or x509 authentication
      - |checkmark|
      - |checkmark|
-     - |checkmark|
-     - |checkmark|
-
-   * - Kerberos, LDAP, and x509 authentication
      - |checkmark|
      -
-     - |checkmark|
-     - |checkmark|
 
    * - :doc:`Schema Analysis </schema>`
      - |checkmark|
-     -
      - |checkmark|
      - |checkmark|
+     -
 
    * - :doc:`Real Time Server Stats </performance>`
      - |checkmark|
-     -
      - |checkmark|
      - |checkmark|
+     -
 
    * - Automatic updates
      - |checkmark|
+     - 
      - |checkmark|
      - |checkmark|
-     -
 
    * - Telemetry data collection via
        `Stitch <https://www.mongodb.com/cloud/stitch>`_
      - |checkmark|
+     - 
      - |checkmark|
      - |checkmark|
-     -
 
    * - Map rendering in the :ref:`Schema <schema-tab>` view via third
        party services
      - |checkmark|
-     -
+     - 
      - |checkmark|
      -
 
    * - Error collection and crash reporting
      - |checkmark|
+     - 
      - |checkmark|
      - |checkmark|
-     -
 
 Each |compass| edition is available in both a
 *General Availability* and a *Beta* version. To download any
```

Comment:
What does this actually mean? What is the actual difference here? Should be up in the Editions section.